web/public/templates/dojos/events/user-events.dust

```diff
@@ -24,7 +24,7 @@
             <tr ng-click="showEventInfo(dojoEvents, $index)" class="event-info-row">
               <td>{{ event.name }}</td>
               <td>{{ event.formattedDate }}</td>
-              <td>{{ event.type }}</td>
+              <td>{{ getEventTypeDisplayText(event) }}</td>
```

Comment:
Actually, there's a loop on the events already, buried deep inside `user-events-controller.js`. key-val object also makes sense. I still prefer the use of a function over adding an extra property to the event objection, but yeah, one-time binding makes sense here. On all the cells really.